libqtile/widget/backlight.py

```diff
@@ -102,12 +103,17 @@ def __init__(self, **config):
             self.max_brightness_file,
         )
 
-        self.add_callbacks(
-            {
+        if self.invert_mouse_wheel:
+            callbacks = {
+                "Button4": partial(self.cmd_change_backlight, ChangeDirection.DOWN),
+                "Button5": partial(self.cmd_change_backlight, ChangeDirection.UP),
+            }
+        else:
+            callbacks = {
                 "Button4": partial(self.cmd_change_backlight, ChangeDirection.UP),
                 "Button5": partial(self.cmd_change_backlight, ChangeDirection.DOWN),
             }
-        )
+        self.add_callbacks(callbacks)
```

Comment:
I have a slight preference for writing it like this but don't feel particularly strongly about it!
``` python
self.add_callbacks(
    {
        "Button4": partial(self.cmd_change_backlight, ChangeDirection.DOWN if self.invert_mouse_wheel else ChangeDirection.UP),
        "Button5": partial(self.cmd_change_backlight, ChangeDirection.UP if self.invert_mouse_wheel else ChangeDirection.DOWN),
    }
)
```

What do you think?

`black` may also format my lines if they're too long.